.github/PULL_REQUEST_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,20 @@
+
+[//]: # (Lines starting with [//]: are considered to be comments. You do not have to delete those lines, they are not rendered upon release.)
+
+**Related User Story**: #
```

Comment:
Add something like "Please provide the id of the user story of the task you implemented. This should be used in order to get information about the user story itself."